scripts/install-conda.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+
+CONDA_INSTALL_PREFIX=/opt/conda
+CONDA_INSTALLER_VERSION=4.12.0-0
```

Comment:
Can we just point to the official docs for installing conda instead?

Reply:
I need this script for CI. In the docs, I point to the official documentation.

Reply:
Perhaps this script should be in `.github/scripts` if it is intended only for CI?

Reply:
I think some users will like using this, but sure I can move it to the `.github/scripts` area.

Reply:
Moved.